qiskit_nature/second_q/operators/sparse_label_op.py

```diff
@@ -121,28 +151,22 @@ def _validate_polynomial_tensor_key(cls, keys: Collection[str]) -> None:
         """
 
     @classmethod
-    def zero(cls, register_length: int | None = None) -> SparseLabelOp:
+    def zero(cls) -> SparseLabelOp:
         """Constructs a zero-operator.
 
-        Args:
-            register_length: the length of the operator.
-
         Returns:
             The zero-operator of the given length.
         """
-        return cls({}, register_length=register_length, copy=False)
+        return cls({}, copy=False)
```

Comment:
Is this always ok, for subclasses (same for `one` below) - this suggests all other parameters they might have are defaulted for all subclasses - we could state that as a requirement perhaps.

Reply:
I don't see a better way honestly :thinking:
I think if a subclass does not support this, it will need to overwrite this method.